### ISSUE TITLE
Using <form>.submit() instead of <form>.live('submit') when calling $('form').address()

### ISSUE THREAD

**Issue by theartoflogic** [asual/jquery-address#38]
Right now it properly uses <link>.live('click') to make sure dynamically loaded links are also handled by the jQuery Address plugin, but forms do not.

Can you change it from using .submit() to .live('submit', fnc) instead so that dynamically-loaded form will be caught by the plugin as well?

Note: This only applies when using $('form').address(). To get around this issue you can do the following:

```
    $('form').live('submit', function(e){
        e.preventDefault();
        var url = $(this).attr('action');
        url += (url.indexOf('?') == -1) ? '?' : (url.indexOf('&') != (url.length-1) ? '&' : '');
        url += $(this).serialize();
        $.address.value(url);
    });
```

**Comment by asual** [comment]
Originally the plugin was using .live() but later I discovered that it's not working in IE8. Can you try it and confirm? If the behavior has changed in jQuery 1.4.4 it will be great to use .live() again.

**Comment by theartoflogic** [comment]
According to this bug http://bugs.jquery.com/ticket/6359 it has been fixed. That was only two months ago, so it has probably not made it into an official release yet.

However, there is an easy workaround for the bug until a new version of jQuery is released, as stated on that bug report:

```
"However, if you attach the submit handler *before* the click handler, it works."
```

I tested this and it does indeed work. Now, instead of me doing:

```
$('a').address();
$('form').live('submit');
```

...I do...

```
$('form').live('submit');
$('a').address();
```

If you change your plugin to switch the order in which you apply the event handlers so that the form submit handler is defined first then it should fix this problem and make using $('form').address() work for dynamically-generated forms as well.
